Makefile

```diff
@@ -100,7 +100,6 @@ checks:  check-duplicate-error-messages
 # check-duplicate-error-messages will conform with 
 .PHONY: check-duplicate-error-messages
 check-duplicate-error-messages:
-	@(test $$(grep -Ir errors.New . | grep -v -e .*.md | wc -l) -eq 0) || (echo "please use fmt.Errorf and not errors.New" >&2 && exit 1)
```

Comment:
I don't know why we have this check. It makes no sense. There is no reason not to use `errors.New()`, it is the normal way to create unformatted errors.